Jenkinsfile

```diff
@@ -313,6 +313,41 @@ def buildHipClangJobAndReboot(Map conf=[:]){
     }
 }
 
+def RunPerfTest(Map conf=[:]){
+    def dockerOpts="--device=/dev/kfd --device=/dev/dri --group-add video --group-add render --cap-add=SYS_PTRACE --security-opt seccomp=unconfined"
+    try {
+        (retimage, image) = getDockerImage(conf)
+        withDockerContainer(image: image, args: dockerOpts + ' -v=/var/jenkins/:/var/jenkins') {
+        timeout(time: 600, unit: 'MINUTES')
+            {
+                unstash 'miopen_tar'
+                sh "tar -zxvf build/miopen-hip-*-Linux-runtime.tar.gz"
+                ld_lib="${env.WORKSPACE}/opt/rocm/lib"
+                def filename = conf.get("filename", "Resnet50_v1.5.txt") // default Resnet50_v1.5 
+                sh "export LD_LIBRARY_PATH=${ld_lib} && ${env.WORKSPACE}/opt/rocm/bin/test_perf.py  --filename ${filename} --install_path ${env.WORKSPACE}/opt/rocm"
+                jenkins_url = "${env.artifact_path}/${env.MIOPEN_PERF_BRANCH}/lastSuccessfulBuild/artifact" 
+                try {
+                    sh "wget -P ${env.WORKSPACE}/opt/rocm/bin/old_results/ ${jenkins_url}/opt/rocm/bin/perf_results/${filename}"
+                } catch (Exception err){
+                    currentBuild.result = 'SUCCESS'
+                    }
```

Comment:
Wrong indentation

Reply:
Resolved.

@alexandraBara Please hide resolved comments (I do not have permissions)